README.md

```diff
@@ -51,7 +51,12 @@ module.exports = function() {
       ua: 'foo',
       callback: 'bar',
       unknown: 'baz',
-      rum: 0
+      rum: 0,
+
+      // Configure where the polyfill script will be added to the index.html file.
+      // Default is `head` ember-cli provides `head`, `head-footer`, `body` and `body-footer`
+      // FYI you need to load the polyfill before your app.js file
```

Comment:
What does this mean exactly? Just that if your `app.js` is in the `body-footer`, that the polyfill must also?

Reply:
I saying that any polyfills the dev add to their app needs to be loaded before the `app.js` file. In the test case it would be `dummy.js`

`<script src="{{rootURL}}assets/dummy.js"></script>`

If the dev moves the polyfill to the footer of the index.html the they leave the `vendor.js` and `app.js` in the head the dev might encounter some errors

Is that clearer?

https://www.outdoorsy.com
using this example as of April 25 2018 we are loading the polyfill in the head which is blocking render of the server side render or fastboot render.

Reply:
> I saying that any polyfills the dev add to their app needs to be loaded before the app.js file.

Right. What does FastBoot do with the `dummy.js`? Is it also using `body-footer` to insert the script at the right location? Is there a way to guarantee that the ordering is correct?

Reply:
you need your js scripts to be loaded after the content to insure the page loads fast. 

There is no way to insure the correct order.

Reply:
> you need your js scripts to be loaded after the content to insure the page loads fast.

Right, I understand this part. I'm just not sure if it's even worth mentioning to the user if they can't actually do anything to ensure that it works correctly.

I think it would be better to detect FastBoot (I think there should be a way to do this) and to just inject into `body-footer` in those cases rather than having the user able to configure this manually.